plugins/setting-resources/src/components/PluginCard.svelte

```diff
@@ -38,7 +38,9 @@
 
   async function disconnect(): Promise<void> {
```

Comment:
May be we should remove `close` and `disconnect` functions here, if we create and remove setting using external service